app/components/post_draft/send_handler/__tests__/send_handler.test.tsx

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {act, fireEvent, waitFor} from '@testing-library/react-native';
+import React from 'react';
+
+import {General} from '@constants';
+import {DRAFT_TYPE_DRAFT, DRAFT_TYPE_SCHEDULED, type DraftType} from '@screens/global_drafts/constants';
+import {renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+import {sendMessageWithAlert} from '@utils/post';
+
+import SendHandler from '../send_handler';
+
+import type {Database} from '@nozbe/watermelondb';
+
+jest.mock('@actions/remote/channel', () => ({
+    getChannelTimezones: jest.fn().mockResolvedValue({channelTimezones: []}),
+}));
+
+jest.mock('@utils/post', () => ({
+    sendMessageWithAlert: jest.fn(),
+}));
+
+jest.mock('@screens/navigation', () => ({
+    dismissBottomSheet: jest.fn(),
+}));
+
+describe('components/post_draft/send_handler/SendHandler', () => {
+    let database: Database;
+    const baseProps = {
+        testID: 'test_send_handler',
+        channelId: 'channel-id',
+        channelType: General.OPEN_CHANNEL,
+        channelName: 'test-channel',
+        rootId: '',
+        currentUserId: 'current-user-id',
+        cursorPosition: 0,
+        enableConfirmNotificationsToChannel: true,
+        maxMessageLength: 4000,
+        membersCount: 3,
+        useChannelMentions: true,
+        userIsOutOfOffice: false,
+        customEmojis: [],
+        value: '',
+        files: [],
+        clearDraft: jest.fn(),
+        updateValue: jest.fn(),
+        updateCursorPosition: jest.fn(),
+        updatePostInputTop: jest.fn(),
+        addFiles: jest.fn(),
+        uploadFileError: null,
+        setIsFocused: jest.fn(),
+        persistentNotificationInterval: 0,
+        persistentNotificationMaxRecipients: 5,
+        postPriority: {
+            priority: 'urgent',
+            requested_ack: false,
+            persistent_notifications: false,
+        } as PostPriority,
+    };
+
+    beforeAll(async () => {
+        const server = await TestHelper.setupServerDatabase();
+        database = server.database;
+    });
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should match snapshot', () => {
+        const wrapper = renderWithEverything(
+            <SendHandler {...baseProps}/>, {database},
+        );
+        expect(wrapper.toJSON()).toBeTruthy();
+    });
```

Comment:
This is not matching any snapshot, and not sure how useful it is.

Reply:
Yup, that correct, better to remove this from the test.